week_1/Array/sort_colors/sort_colors.js

```diff
@@ -0,0 +1,37 @@
+// Given an array nums with n objects colored red, white, or blue, sort them in-place so that objects of the same color are adjacent, with the colors in the order red, white, and blue.
+
+// We will use the integers 0, 1, and 2 to represent the color red, white, and blue, respectively.
+// You may not use the sort method.
+
+// Warning: This algorithm mutates the input array.
+/**
+ * @param {number[]} nums
+ * @return {void} Do not return anything, modify nums in-place instead.
+ */
+const sortColors = (nums) => {
```

Comment:
Variables should always be as descriptive as possible. Here, nums is a bit misleading. `colors` would be more appropriate.